app/src/main/java/com/greenfox/tribesoflagopusandroid/api/model/gameobject/Building.java

```diff
@@ -6,58 +6,70 @@
  * Created by User on 2017. 06. 14..
  */
 
-public class Building extends BaseResponse{
-
-    private long id;
-    private String type;
-    private int level;
-    private int hp;
-
-    public Building() {
-    }
-
-    public Building(long id, String type, int level, int hp) {
-        this.id = id;
-        this.type = type;
-        this.level = level;
-        this.hp = hp;
-    }
-
-    public Building(String type) {
-        this.type = type;
-    }
-
-    public String getType() {
-        return type;
-    }
-
-    public void setType(String type) {
-        this.type = type;
-    }
-
-    public int getLevel() {
-        return level;
-    }
-
-    public void setLevel(int level) {
-        this.level = level;
-    }
-
-    public int getHp() {
-        return hp;
-    }
-
-    public void setHp(int hp) {
-        this.hp = hp;
-    }
-
-    public long getId() {
-
-        return id;
-    }
-
-    public void setId(long id) {
-        this.id = id;
-    }
+public class Building extends BaseResponse {
+
+  private long id;
+  private String type;
+  private int level;
+  private int hp;
+  private long started_at;
```

Comment:
@rekagiday jackson converts automagically between json style naming and java style naming, hence `startedAt` will be `started_at`, so use standard java naming for that